syncer/client.go

```diff
@@ -0,0 +1,277 @@
+package syncer
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/blockchain"
+	"github.com/0xPolygon/polygon-edge/network"
+	"github.com/0xPolygon/polygon-edge/network/event"
+	"github.com/0xPolygon/polygon-edge/syncer/proto"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+const (
+	SyncPeersManagerLoggerName = "sync-peers-manager"
+	statusTopicName            = "syncer/status/0.1"
+	defaultTimeoutForStatus    = 10 * time.Second
+)
+
+type syncPeerClient struct {
+	logger     hclog.Logger
+	network    Network
+	blockchain Blockchain
+
+	subscription           blockchain.Subscription
+	topic                  *network.Topic
+	id                     string // node ID
+	peerStatusUpdateCh     chan *NoForkPeer
+	peerConnectionUpdateCh chan *event.PeerEvent
+}
+
+func NewSyncPeerClient(
+	logger hclog.Logger,
+	network Network,
+	blockchain Blockchain,
+) SyncPeerClient {
+	return &syncPeerClient{
+		logger:                 logger.Named(SyncPeersManagerLoggerName),
```

Comment:
You are creating a `SyncPeerClient`, but the logger is named `sync-peers-manager`

Reply:
Sorry, I forgot to rename. Fixed at 85c8b684c6bbbff7f3e604b870668149a2a1bd0a